### PR TITLE
The old pager is now removed

### PR DESCRIPTION
**Pull request by annando**
The old pager is removed. The old pager was really slow and it complicated the whole pager thing.

### CODE REVIEW

File: include/Contact.php [hunk 1]
```diff
@@ -726,11 +717,7 @@ function posts_from_gcontact(App $a, $gcontact_id) {
 
 	$o = conversation($a,$r,'community',false);
 
-	if(!get_config('system', 'old_pager')) {
-		$o .= alt_pager($a,count($r));
-	} else {
-		$o .= paginate($a);
-	}
+	$o .= alt_pager($a,count($r));
```
Comment (Hypolite) [inline include/Contact.php]: Standards: Can you please add a space after the comma?

File: include/Contact.php [hunk 2]
```diff
@@ -782,11 +760,7 @@ function posts_from_contact_url(App $a, $contact_url) {
 
 	$o = conversation($a,$r,'community',false);
 
-	if (!get_config('system', 'old_pager')) {
-		$o .= alt_pager($a,count($r));
-	} else {
-		$o .= paginate($a);
-	}
+	$o .= alt_pager($a,count($r));
```
Comment (Hypolite) [inline include/Contact.php]: Standards: Same here.

Comment (Hypolite) [inline include/Contact.php]: Standards: Same here.

**Comment by annando** [comment]
@Hypolite Done.

File: mod/community.php [hunk 3]
```diff
@@ -107,11 +86,7 @@ function community_content(App $a, $update = 0) {
 
 	$o .= conversation($a,$s,'community',$update);
 
-	if(!get_config('system', 'old_pager')) {
-	        $o .= alt_pager($a,count($r));
-	} else {
-	        $o .= paginate($a);
-	}
+        $o .= alt_pager($a,count($r));
```
Comment (Hypolite) [inline mod/community.php]: Sorry to bother you again, line comments don't work well on phone. Here's a new instance of missing space.

File: mod/network.php [hunk 4]
```diff
@@ -793,10 +778,8 @@ function network_content(App $a, $update = 0) {
 	if (!$update) {
 		if (get_pconfig(local_user(),'system','infinite_scroll')) {
 			$o .= scroll_loader();
-		} elseif (!get_config('system', 'old_pager')) {
-			$o .= alt_pager($a,count($items));
 		} else {
-			$o .= paginate($a);
+			$o .= alt_pager($a,count($items));
```
Comment (Hypolite) [inline mod/network.php]: Another one.

File: mod/profile.php [hunk 5]
```diff
@@ -337,12 +320,8 @@ function profile_content(App $a, $update = 0) {
...
 	$o .= conversation($a,$items,'profile',$update);
 
-	if(! $update) {
-		if(!get_config('system', 'old_pager')) {
-			$o .= alt_pager($a,count($items));
-		} else {
-			$o .= paginate($a);
-		}
+	if (!$update) {
+		$o .= alt_pager($a,count($items));
```
Comment (Hypolite) [inline mod/profile.php]: And this is the last one.

**Comment by annando** [comment]
@Hypolite done again ;-)

**Comment by annando** [comment]
@tobiasd do you have time to accept it?

**Pull request merged**